server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  content: [
+    {
+      _id: Schema.Types.ObjectId,
+      type: Schema.Types.Mixed,
```

Comment:
Why two types?

Reply:
Right! So type should be a string desribing the type of the inner content here. I'll update that